### DISCUSSION TITLE
Suddenly they all stopped.

### DISCUSSION THREAD

**Discussion by littledog75** [dtcooper/raspotify#560] [Q&A] [answered]
First! Thanks for this great application. I have been using it for years and it works great! 

I'm rocking three of these raspotify units at home. all on different generations of raspberries. 
Suddenly this Tuesday, they all stopped working. It connects, but doesn't play any sound. They also kind of fail to pause and register song length. And all of them got the same issue at once.

Looking at the systemd logs I can find this on all three.
[2022-07-20T13:36:44Z ERROR librespot_playback::player] Player Commands Error: channel closed

Am I the only one that has this issue?

I really love this setup and will be very grateful for any ideas.

**Comment by kingcharles1666** [comment]
I have the same problem, I tried rebooting the pi and restarting the application and sometimes it would play something but not what I selected ?
Really weird, maybe it is something with spotify?

**Comment by JasonLG1979** [comment]
Channel errors are errors on Spotify's side. There is nothing we can do about the really.

https://github.com/librespot-org/librespot/issues/972

**Comment by littledog75** [comment, marked as answer]
Thanks for the tips. Found a workaround.
Add this to /etc/hosts
104.199.65.124 ap-gew4.spotify.com

Awesome! Thank you.

**Reply by JasonLG1979** [reply]
This is no longer needed. Librespot 0.4.2 blacklists that AP.

**Comment by TupaNegreiros** [comment]
I'm having this problem too, Spotify player keeps skipping songs, librespot says it's "Unable to load encrypted file".
Looks like librespot version 0.4.2 solves this issue #972. What is needed to update librespot installed by raspotify?

**Reply by JasonLG1979** [reply]
If you've added the repo all you need to do is update your system as usual. The latest version of Raspotify comes with librespot 0.4.2.

**Reply by riwid** [reply]
I am having the same issue but and update doesn't update to librespot version 0.4.2.
```
apt list --upgradable -a   
Listing... Done
raspotify/unknown 0.42.0~librespot.v0.4.2-6537c441 armhf [upgradable from: 0.31.3~librespot.v0.3.1-19-gbbd575e]  
raspotify/now 0.31.3~librespot.v0.3.1-19-gbbd575e armhf [installed,upgradable to: 0.42.0~librespot.v0.4.2-6537c441]
```

But when I run apt update the package is not updated.

**Reply by TupaNegreiros** [reply]
I was unable to update raspotify/librespot in Raspbian Buster, because of dependency of other packages. I had to upgrade to Raspbian Bullseye.

**Reply by JasonLG1979** [reply]
I'm sorry update, upgrade. Yep, the addition of the pulseaudio backend added libpulse as a dependency.
